benchmark/python/benchmark_e2e.py

```diff
@@ -83,6 +83,14 @@ def generate_prompt(model, tokenizer, prompt_length, use_graph_capture) -> str:
         generator.generate_next_token()
     return tokenizer.decode(generator.get_sequence(0))
 
+# Use prompt length to get pre-defined prompt
+def get_prompt_by_length(prompt_length):
+    json_path = "prompts.json"
```

Comment:
Instead of uploading another copy of `prompts.json`, can we download it from [here](https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/python/tools/transformers/models/llama/prompts.json) and save it to disk using `requests` or `urllib` instead? That way, only one location has to be updated when adding other prompt lengths.